mon-pix/tests/integration/components/navbar-burger-menu-test.js

```diff
@@ -12,27 +12,46 @@ describe('Integration | Component | navbar-burger-menu', function() {
 
   beforeEach(async function() {
     class currentUser extends Service { user = {
-      email: 'bobby.carotte@example.net',
       fullName: 'Bobby Carotte',
     }}
     this.owner.register('service:currentUser', currentUser);
 
   });
 
+  it('should display the user\'s fullname', async function() {
+    // when
+    await render(hbs`<NavbarBurgerMenu />`);
+
+    // then
+    expect(contains('Bobby Carotte')).to.exist;
+  });
+
   it('should display the navigation menu with expected elements outside of campaign results', async function() {
     // when
     await render(hbs`<NavbarBurgerMenu />`);
 
     // then
     expect(find('.navbar-burger-menu__navigation')).to.exist;
 
-    expect(findAll('.navbar-burger-menu-navigation__item')).to.have.lengthOf(6);
-    expect(contains('Accueil')).to.exist;
-    expect(contains('Mon compte')).to.exist;
-    expect(contains('Compétences')).to.exist;
-    expect(contains('Certification')).to.exist;
-    expect(contains('Mes tutos')).to.exist;
-    expect(contains('J\'ai un code')).to.exist;
+    expect(findAll('.navbar-burger-menu-navigation__item')).to.have.lengthOf(5);
+    expect(contains(this.intl.t('navigation.main.dashboard'))).to.exist;
+    expect(contains(this.intl.t('navigation.main.skills'))).to.exist;
+    expect(contains(this.intl.t('navigation.main.start-certification'))).to.exist;
+    expect(contains(this.intl.t('navigation.main.tutorials'))).to.exist;
+    expect(contains(this.intl.t('navigation.main.code'))).to.exist;
+  });
+
+  it('should display the user menu with expected elements', async function() {
```

Comment:
**suggestion**
Je trouve bizarre ces deux  `it` : ils ont quasiment le même intitulé mais ne testent pas du tout la même partie.
- On pourrait peut être faire 2 contextes pour aider à faire la distinction ? (2 contextes qui pourraient permettre d'ordonner tous les autres tests du fichier je pense)
- Et détailler un peu plus ce qu'on entends par `expected elements` ?

``` 
context('upper part of the menu', () => {
    it('should display the user\'s fullname')
    it('should display the navigation menu with dashboard, skills, start-certification, tutorials and code')
})

context('lower part of the menu', () => {
      it('should display the user menu with account, certification, help, sign-out informations')
      context('when user has participations') 
      // ... etc
})
```